### PR TITLE
Fixes #96 CSS transitions

### LINKED ISSUE #96: The image CSS visibility transition overruns
Version 3 of IIZ.

The image visibility transition overruns.

The transition takes twice the fadeDuration period, whereas It should probably take just the fadeDuration period.

This is best understood by looking at the screenshot.

This was taken with the Chrome dev tools and relates to the zoom in sequence.
The zoom out is correct, and is not shown.

Visibility overrun

I've tried out a fix for this that seems to work.

### PR DESCRIPTION
**Pull request by MaxdAyala**

### CODE REVIEW

File: .github/workflows/test-ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Run unit tests on ubuntu-latest
```
Comment (laurenashpole) [inline .github/workflows/test-ubuntu.yml:1]: The release action for this repo runs tests on `ubuntu-latest`, `macos-latest`, `windows-latest` so a separate file isn't necessary.

File: src/InnerImageZoom/components/Image.js [hunk 2]
```diff
@@ -18,9 +18,9 @@ const Image = ({ src, sources, width, height, hasSpacer, imgAttributes, isZoomed
               createSpacer ? 'iiz__img--abs' : ''
             }`}
             style={{
-              transition: `linear 0ms opacity ${isZoomed ? fadeDuration : 0}ms, linear ${fadeDuration}ms visibility ${
-                isZoomed ? fadeDuration : 0
-              }ms`
+              transition:
+                `opacity 0ms linear ${isZoomed ? fadeDuration : 0}ms, ` +
+                `visibility ${isZoomed ? 0 : fadeDuration}ms linear ${isZoomed ? fadeDuration : 0}ms`
```
Comment (laurenashpole) [inline src/InnerImageZoom/components/Image.js:23]: I really had to think back on the original intention for this image. The image should disappear after the zoom image has transitioned in without any effect on itself. It should reappear immediately before the zoom image begins transitioning out, again with no effect.

The overrun you were seeing looks like it was caused by the transition duration on visibility. So this line should probably look something like this instead:

```
`visibility 0ms linear ${isZoomed ? fadeDuration : 0}ms`
```

File: src/InnerImageZoom/components/Image.js [hunk 3]
```diff
@@ -34,9 +34,9 @@ const Image = ({ src, sources, width, height, hasSpacer, imgAttributes, isZoomed
             createSpacer ? 'iiz__img--abs' : ''
           }`}
           style={{
-            transition: `linear 0ms opacity ${isZoomed ? fadeDuration : 0}ms, linear ${fadeDuration}ms visibility ${
-              isZoomed ? fadeDuration : 0
-            }ms`
+            transition:
+              `opacity 0ms linear ${isZoomed ? fadeDuration : 0}ms, ` +
+              `visibility ${isZoomed ? 0 : fadeDuration}ms linear ${isZoomed ? fadeDuration : 0}ms`
```
Comment (laurenashpole) [inline src/InnerImageZoom/components/Image.js:39]: This line should also be changed to match the comment above.

**Review by laurenashpole — CHANGES_REQUESTED** [review]
Thanks again for opening a PR! And for noticing that extra animation time. I left a comment based on the original intention for the main images.

**Comment by MaxdAyala** [comment]
Changes made, checks passing.

**Pull request merged**